src/Compilers/CSharp/Portable/Symbols/Source/ExplicitInterfaceHelpers.cs

```diff
@@ -17,21 +17,78 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
+#nullable enable
+    internal class ExplicitInterfaceMemberInfo(Binder binder, BindingDiagnosticBag diagnostics, ExplicitInterfaceSpecifierSyntax explicitInterfaceSpecifier, string name, string? aliasQualifier)
```

Comment:
>ExplicitInterfaceMemberInfo [](http://example.com/codeflow?start=19&length=27)

Is this meant to be stored in a symbol? In general, symbols shouldn't be holding onto syntax nodes and binders